frontend/src/modules/AppWrapper.tsx

```diff
@@ -19,60 +20,75 @@ import {
 } from "Src/pages";
 import { RouteProvider } from "./RouteProvider";
 
+function ErrorFallback({ error }) {
+	return (
+		<Row justify="space-around" align="middle" style={{ margin: 200 }}>
```

Comment:
it would be good if we can wrap this row component with styled component and there we can go with margin


but why there is a 200 margin. can we just centre the div?

Reply:
Ya sure @palashgdev i will the style as styled component and about margin i will try and let u konw

Reply:
i have tried with out margin:200px the Error Box in the top center Position

![Screenshot_2021-07-29 Open source Observability platform SigNoz](https://user-images.githubusercontent.com/8166149/127520970-23ece319-b4d0-4d07-93b2-17f4a5ec720a.png)

Thats  why i have added the margin 100 is there any other Fix @palashgdev  man